pkg/api/meta/meta.go

```diff
@@ -29,8 +30,18 @@ import (
 // obj must be a pointer to an API type. An error is returned if the minimum
 // required fields are missing. Fields that are not required return the default
 // value and are a no-op if set.
-// TODO: add a fast path for *TypeMeta and *ObjectMeta for internal objects
-func Accessor(obj interface{}) (Interface, error) {
+func Accessor(obj interface{}) (Object, error) {
```

Comment:
Missed this before.  Why the signature change?  Because `Object` now has direct access to these methods?  If so, I'm fine with it, but the name is odd.